### ISSUE TITLE
delete this repo?

### ISSUE THREAD

**Issue by sappelhoff** [bids-standard/bids2datapackage#1]
@franklin-feingold maybe you can add this repo to your list of "to-be-deleted" repos.

It seems to me like no work was done here and as such, it just clutters the Org?

If I have a wrong impression, please speak up (whoever reads this)!

**Comment by franklin-feingold** [comment]
yep it looks pretty stale. The data package that this repo references may be superseded by our BIDS-schema (https://github.com/bids-standard/bids-schema - related: https://github.com/bids-standard/bids-specification/pull/475#issuecomment-663137965
